src/rtnl/tc/constants.rs

```diff
@@ -93,3 +93,12 @@ pub const TCA_EGRESS_REDIR: i32 = 1; /* packet redirect to EGRESS */
 pub const TCA_EGRESS_MIRROR: i32 = 2; /* mirror packet to EGRESS */
 pub const TCA_INGRESS_REDIR: i32 = 3; /* packet redirect to INGRESS */
 pub const TCA_INGRESS_MIRROR: i32 = 4; /* mirror packet to INGRESS */
+
+/// NAT action attr
+pub const TCA_NAT_UNSPEC: u16 = 0;
+pub const TCA_NAT_PARMS: u16 = 1;
+pub const TCA_NAT_TM: u16 = 2;
+pub const TCA_NAT_PAD: u16 = 3;
+pub const TCA_NAT_MAX: u16 = TCA_NAT_PAD;
```

Comment:
Please remove `TCA_NAT_PAD` and `TCA_NAT_MAX`.
The `TCA_NAT_PAD` is unused anywhere. The `TCA_NAT_MAX` is not constant to kernel but a internal variable might change in future release of linux kernel.